### PR TITLE
A11y updates (remove placeholder)

### PR DESCRIPTION
**Pull request by lynnco**

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 engines:
   duplication:
     enabled: true
+    exclude_fingerprints:
+    - 871d85232f90c108abf687a789b06606
```
Comment (kelseyq) [inline .codeclimate.yml]: what's this exclude for?

Comment (lynnco) [inline .codeclimate.yml]: @kelseyq https://codeclimate.com/github/SexualHealthInnovations/django-wizard-builder/wizard_builder/models.py#issue_5819027bde98af0001000024

(link was in the description, was easy to miss)

Comment (kelseyq) [inline .codeclimate.yml]: hmmm...should we exclude it if it's a real issue? the code climate check doesn't block builds from merging

Comment (lynnco) [inline .codeclimate.yml]: Oh! I didn't know it didn't block merges. I'll push w/o this change

Comment (lynnco) [inline .codeclimate.yml]: Done 👍

**Pull request merged**